--- mercurial-1.1.2/contrib/mergetools.hgrc.orig	2009-02-07 07:14:46.000000000 -0500
+++ mercurial-1.1.2/contrib/mergetools.hgrc	2009-02-07 07:14:15.000000000 -0500
@@ -46,6 +46,6 @@
 ecmerge.regkey=Software\Elli\xc3\xa9 Computing\Merge
 ecmerge.gui=True
 
-filemerge.executable=/Developer/Applications/Utilities/FileMerge.app/Contents/MacOS/FileMerge
-filemerge.args=-left $other -right $local -ancestor $base -merge $output
-filemerge.gui=True
+#filemerge.executable=/Developer/Applications/Utilities/FileMerge.app/Contents/MacOS/FileMerge
+#filemerge.args=-left $other -right $local -ancestor $base -merge $output
+#filemerge.gui=True
