--- contrib/mergetools.hgrc.orig	2008-03-24 19:05:20.000000000 -0400
+++ contrib/mergetools.hgrc	2008-04-15 15:27:46.000000000 -0400
@@ -1,11 +1,12 @@
 # Some default global settings for common merge tools
 
 [merge-tools]
-kdiff3.args=--auto -L1 base --L2 local --L3 other $base $local $other -o $output
+kdiff3.args=--auto --L1 base --L2 local --L3 other $base $local $other -o $output
 kdiff3.regkey=Software\KDiff3
 kdiff3.regappend=\kdiff3.exe
 kdiff3.fixeol=True
 kdiff3.gui=True
+kdiff3.priority=1
 
 gvimdiff.args=--nofork -d -g -O $local $other $base
 gvimdiff.regkey=Software\Vim\GVim
